<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>613</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-color: rgb(38, 38, 38);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PushButtonCos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonSin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonAsin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonAcos">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonPi">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>pi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonMod">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:   rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:   rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton6">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton7">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:   rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonMult">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonMinus">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>400</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton9">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:   rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:   rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonDot">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>540</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:   rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonLog">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>540</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonPlus">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonLn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonAtan">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonExp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>470</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonTan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonSqrt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>540</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton0">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:   rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonEqual">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>540</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
border-color: rgb(172, 172, 173);
border-width: 1px;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonSign">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonDiv">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>260</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonBracClose">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonX">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(8, 4, 41);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonDel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>DEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonBracOpen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonAC">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(28, 42, 64);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color:rgb(8, 4, 41); 
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox  {
background-color:  rgb(54, 53, 54);
    border: 2px solid darkgray;
    selection-background-color: lightgray;
}
</string>
    </property>
    <item>
     <property name="text">
      <string>          Калькулятор</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Кредитный калькулятор</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Депозитный калькулятор</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_X">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0.000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter X value:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonGraph">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:  rgb(54, 53, 54);
	border-style: outset;
	border-radius: 15px;
	border-color: rgb(33, 33, 33);
}
QPushButton:pressed {
background-color: rgb(39, 39, 39);
}</string>
    </property>
    <property name="text">
     <string>Graphing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Y max</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxY">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
background-color:  rgb(54, 53, 54);
}</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxX">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
background-color:  rgb(54, 53, 54);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> X max</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>701</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
